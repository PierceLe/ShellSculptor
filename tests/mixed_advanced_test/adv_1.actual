>> cd /hahaha
cd: no such file or directory: /hahaha
>> cd ~/
>> pwd
/Users/piercele
>> ls
BreastCancer-TumorStageAnalysis
Desktop
Documents
Downloads
GameTank
Library
Movies
Music
PetDecision-USYD
Pictures
Postman
Public
Test.java
Week 5.pdf
apps
bin
miniconda3
test
test.sh
test.txt
test2.py
>> var hi hello
>> echo ${hi}
hello
>> which cd
cd: shell built-in command
>> which echo
/bin/echo
>> which echo cd var | cat | grep "/usr"
>> exit
