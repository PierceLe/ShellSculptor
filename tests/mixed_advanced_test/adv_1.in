cd /hahaha
cd ~/
pwd
ls
var hi hello
echo ${hi}
which cd
which echo
which echo cd var | cat | grep "/usr"
exit
