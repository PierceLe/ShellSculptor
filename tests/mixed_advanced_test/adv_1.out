>> cd /hahaha
cd: no such file or directory: /hahaha
>> cd ~/
>> pwd
/home
>> ls
cd_command.py
choose_command.py
configure_mysh.py
executing_commands.py
exit_command.py
mysh_command.py
mysh.py
parsing.py
pwd_command.py
__pycache__
README.md
test_data
tests
validate.py
var_command.py
which_command.py
>> var hi hello
>> echo ${hi}
hello
>> which cd
cd: shell built-in command
>> which echo
/usr/sbin/echo
>> which echo cd var | cat | grep "/usr"
/usr/sbin/echo
>> exit
